src/main/java/Listener.java

```diff
@@ -214,8 +222,8 @@ public void onMessageReceived(MessageReceivedEvent event) {
             case "math-help":
                 mathHelp(event);
                 break;
-            case "german";
-                german(event);
+            case "foreign";
+                foreign(event);
```

Comment:
do u know how to write a case for switch case? do u not see the other cases above? can u not copy paste?